captum/attr/_utils/attribution.py

```diff
@@ -287,6 +288,35 @@ def __init__(self, forward_func, layer, device_ids=None):
         """
         super().__init__(forward_func, layer, device_ids)
 
+    def interpolate(layer_attribution, interpolate_dims, interpolate_mode="nearest"):
+        r"""
+        Interpolates given 3D, 4D or 5D layer attribution to given dimensions.
+        This is often utilized to upsample the attribution of a convolutional layer
+        to the size of an input, which allows visualizing in the input space.
+
+        Args:
+
+            layer_attribution (torch.Tensor):  Tensor of given layer attributions.
+            interpolate_dims (int or tuple): Upsampled dimensions. The
+                        number of elements must be the number of dimensions
+                        of layer_attribution - 2, since the first dimension
+                        corresponds to number of examples and the second is
+                        assumed to correspond to the number of channels.
+            interpolate_mode (str):  Method for interpolation, which
+                        muat be a valid input interpolation mode for
+                        torch.nn.functional. These methods are
+                        "nearest", "area", "linear" (3D-only), "bilinear"
+                        (4D-only), "bicubic" (4D-only), "trilinear" (5D-only)
+                        based on the number of dimensions of the given layer
+                        attribution.
+
+        Returns:
+            torch.Tensor:
```

Comment:
We could use it, that's particularly useful when we want to use named returns, since the default structure is to just give a type and description. Here, the return is the upsampled attributions, so I just gave the type and description like the default, but if you prefer, can switch to naming this return!

Reply:
I guess it's more of a convention whether we want to everywhere name our returns or not for public methods. If we have done that everywhere or in most cases it would be good to keep the convention.